### PR TITLE
Add shipping dimensions image

### LINKED ISSUE #34329: [Shipping] Dimensions
**User story**

- As a merchant,
- I want to enter accurate shipping information
- So that I don’t overcharge my customers

**Design**
5sAIeTRd9Yp7nSCT33BAWz-fi-6590%3A293356

**Acceptance criteria**
- [x] Input fields use the Gutenberg unit picker.
- [x] By default, the units would follow the store’s locale/global settings.
- [x] When the length, width, and height fields are focused, we highlight different parts of the illustration on the right. Design: `5sAIeTRd9Yp7nSCT33BAWz-fi-6590%3A293356`.
- [x] We highlight the arrow, dimension name (A/B/C), and part of the illustration.
- [x] There’s no highlight for the weight field.

### PR DESCRIPTION
**Pull request by mdperez86**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Partialy closes https://github.com/woocommerce/woocommerce/issues/34329.

### How to test the changes in this Pull Request:

1. Go to new product page.
2. In Shipping Dimensions section a package image should be shown on the right side of the diemensions fields.
3. If you focus the fields (A, B, C) a particular side of the package image should be highlighted.

<video src="https://user-images.githubusercontent.com/13334210/192798656-9a5bacc3-bc3a-42ef-a450-2ffda8553258.mov"></video>

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm --filter=<project> run changelog add`?

### FOR PR REVIEWER ONLY:

-   [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce-admin/client/products/fields/shipping-dimensions-image/shipping-dimensions-image.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		<svg
+			{ ...props }
+			viewBox="0 0 288 195"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			{ /* Side A */ }
+			<path
+				d="M10.4922 134.221V35.2617C10.4922 33.8539 11.9079 32.8867 13.2193 33.3986L98.3109 66.6076C99.0711 66.9043 99.5748 67.633 99.5837 68.449L100.703 171.089C100.719 172.534 99.2449 173.518 97.9167 172.95L11.7054 136.06C10.9695 135.745 10.4922 135.022 10.4922 134.221Z"
+				fill={ highlight === 'A' ? '#F0F6FC' : '#F6F7F7' }
```
Comment (joshuatf) [inline plugins/woocommerce-admin/client/products/fields/shipping-dimensions-image/shipping-dimensions-image.tsx:19]: Love this really simple solution; nicely done!

File: plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx [hunk 2]
```diff
@@ -54,6 +59,8 @@ function getInterpolatedSizeLabel( mixedString: string ) {
 export const ProductShippingSection: React.FC = () => {
 	const { getInputProps } = useFormContext< Product >();
 	const { formatNumber, parseNumber } = useProductHelper();
+	const [ highlightSide, setHighLightSide ] =
```
Comment (joshuatf) [inline plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx]: Nitpicky, but should be `setHighlightSide`. ("Highlight" being one word)

File: plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx [hunk 3]
```diff
@@ -166,8 +173,8 @@ export const ProductShippingSection: React.FC = () => {
 									'woocommerce'
 								) }
 							</p>
-							<div className="product-shipping-section__container">
-								<div>
+							<div className="product-shipping-section__dimensions__body">
```
Comment (joshuatf) [inline plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx]: We typically use a similar class naming style to [BEM](https://getbem.com/naming/) so this class could either be:

`product-shipping-section__dimensions-body`

Or if dimensions is its own standalone block:

`product-shipping-section-dimensions__body`

**Review by joshuatf — DISMISSED** [review]
This is looking great and testing well!  Left a couple minor nitpicky naming comments, but pre-approving.

File: plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx (hunk 2 above)
Comment (mdperez86) [reply plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx]: Absolutly ;)

File: plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx (hunk 3 above)
Comment (mdperez86) [reply plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx]: Done

**Review by joshuatf — APPROVED** [review]
Looks great!  Thanks for the updates!

**Pull request merged**